modules/ocf_desktop/files/xsession/fix-displays

```diff
@@ -30,8 +30,19 @@ if __name__ == '__main__':
     # |    \------+ \-------/    \------+    \------+  |
     # +------------------------------------------------+
 
+    is_daisy_chaining = any(monitor.endswith('.8') for monitor in monitors)
+    if is_daisy_chaining:
+        sorted_monitors = list(sorted(monitors, reverse=True))
+    else:
+        sorted_monitors = list(sorted(monitors))
```

Comment:
why tho

Reply:
The first monitor in the daisy chain is always the leftmost one, but it also will sort itself as the last one. So we need to reverse it.

Reply:
could you add a comment?